python/common_validations.py

```diff
@@ -0,0 +1,28 @@
+import string_utils
+from parameters_validation import parameter_validation
+import re
+
+
+@parameter_validation
+def is_version(version: str):
+    if version is None or not version:
+        raise ValueError("version should be non-empty and should not be None")
+    if not re.match(r"\d+\.\d+\.\d+$", version):
+        raise ValueError(
+            "version should include 3 levels of versions consists of numbers  separated with dots. e.g: 10.0.1")
+
+
+@parameter_validation
+def is_tag(tag: str):
+    if tag is None or not tag:
```

Comment:
can't we simply check `if not tag:` ?

Reply:
Fixed by https://github.com/citusdata/tools/pull/91/commits/a2e8aef98e41c785588ddb62c2ed0f902bee180d